h/streamer.py

```diff
@@ -655,12 +620,82 @@ def should_send_annotation_event(socket, annotation, event_data):
     return True
 
 
+def process_message(reader, message):
+    """
+    Process a message from the queue.
+
+    Deserializes and distributes messages from the queue to the appropriate
+    handler functions.
+    """
+    if reader.topic == ANNOTATIONS_TOPIC:
+        broadcast_annotation_message(message, WebSocket.instances)
+    elif reader.topic == USER_TOPIC:
+        broadcast_session_change_message(message, WebSocket.instances)
```

Comment:
Do we assume reader.topic will never be anything other than these two? Should we crash or log something if it is?